superset/db_engine_specs/ocient.py

```diff
@@ -154,6 +160,12 @@ class OcientEngineSpec(BaseEngineSpec):
     force_column_alias_quotes = True
     max_column_name_length = 30
 
+    # Store mapping of superset Query id -> Ocient ID
+    # These are inserted into the cache when executing the query
+    # They are then removed, either upon cancellation or query completion
+    query_id_mapping: Dict[Query, pyocient.Cursor]= dict()
```

Comment:
formatting...

Reply:
And why are we still storing the cursor??

Reply:
Oh maybe this is just a stale type annotation?
``` suggestion
    query_id_mapping: Dict[uuid.UUID, uuid.UUID] = dict()
```

Reply:
Stale type annotation, but UUID is not correct. Will update the types